dnsapi/dns_selectel.sh

```diff
@@ -104,7 +104,7 @@ dns_selectel_rm() {
 # _domain=domain.com
 # _domain_id=sdjkglgdfewsdfg
 _get_root() {
-  domain=$1
+  domain="$(echo "$1" | idn -u)"
```

Comment:
do not use command `idn`,  please use `_idn()` function instead.

Reply:
Replaced `echo "$domain" | idn` to `_idn "$domain"`, tested.
But `idn -u` converts domain from punycode to utf, `_idn` make utf -> punycode convert, I cannot replace it.